tensorflow/core/kernels/BUILD

```diff
@@ -7771,12 +7771,7 @@ tf_mkl_kernel_library(
     hdrs = ["no_op.h"],
```

Comment:
I don't think we need `no_op.h` in `hdrs` anymore now that we have `:no_op` in `dep`. Could you please try if removing it works?

Reply:
Done